DOC-3364: OS 4.5 link fixes (#312)
corda/corda-docs-portal: content/en/platform/corda/4.5/open-source/shell.md

When a shell is running in unsafe mode, the shell behaviour will be the same as before and will include CRaSH built-in commands. By default the internal shell will run in safe mode but will still be have the ability to execute RPC client calls as before based on existing RPC permissions. No Corda functionality is affected by this change; only the ability to access to the CRaSH shell embedded commands.

> When running an SSH shell, it will run in safe mode for any user that does not explicitly have permission ‘ALL’ as one the items in their RPC permission list, see [Working with the CordaRPCClient API](tutorial-clientrpc-api.md) for more information about the RPC Client API. These shell changes arealso applied to the Stand Alone shell which will now run in safe mode (Enterprise 4.3 onwards). It may be possible that, in the future, the CRaSH shell embedded commands may become deprecated. Where possible, please do not write any new code that depends on them as they are technically not part of Corda functionality.

Reviewer: ```suggestion
When running an SSH shell, it will run in safe mode for any user that does not explicitly have permission ‘ALL’ as one the items in their RPC permission list, see [Using the client RPC API](tutorial-clientrpc-api.md) for more information about the RPC Client API. These shell changes are also applied to the Stand Alone shell which will now run in safe mode (Enterprise 4.3 onwards). It may be possible that, in the future, the CRaSH shell embedded commands may become deprecated. Where possible, please do not write any new code that depends on them as they are technically not part of Corda functionality.
```

### Upload and download attachments

> The shell can be used to upload and download attachments from the node. To learn how, see the [Working with attachments](tutorial-attachments.html#using-attachments) tutorial.

Reviewer: ```suggestion
The shell can be used to upload and download attachments from the node. To learn how, see the [Using attachments](tutorial-attachments.html#using-attachments) tutorial.
```
